owlbot.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script is used to synthesize generated parts of this library."""
+
+import pathlib
+
+import synthtool as s
+from synthtool import gcp
+from synthtool.languages import python
+
+REPO_ROOT = pathlib.Path(__file__).parent.absolute()
+
+common = gcp.CommonTemplates()
+
+# ----------------------------------------------------------------------------
+# Add templated files
+# ----------------------------------------------------------------------------
+
+extras = ["tqdm"]
+templated_files = common.py_library(
+    unit_test_python_versions=["3.7", "3.8", "3.9"],
+    system_test_python_versions=["3.8", "3.9"],
+    cov_level=73,
```

Comment:
Ugh, that seems awfully low. :(

Reply:
Agreed. Filed https://github.com/googleapis/python-bigquery-pandas/issues/392 to get this up. It's one of the reasons I haven't called `pandas-gbq` 1.0 yet, despite it being relatively stable.

Reply:
Locally I get 86%. I'll see if I get the same on Kokoro.